cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -243,6 +286,9 @@ public void eventReceived(Action action, KafkaConnector kafkaConnector) {
                                                 + Labels.STRIMZI_CLUSTER_LABEL
                                                 + "': No connect cluster in which to create this connector."),
                                         kafkaConnector, connectOperator.connectorOperator);
+                                if (isPaused(kafkaConnector.getStatus())) {
+                                    connectOperator.pausedConnectorsResourceCounter(connectorNamespace).decrementAndGet();
+                                }
```

Comment:
Same as with the two comments at the start of the event watch => and explanatory comment will make sure everyone better understands the logic in the future.